### PR TITLE
fix dashboard images

### PR DESCRIPTION
**Pull request by mazmassa**

### CODE REVIEW

**Review by lilyanB — APPROVED** [review]

**Comment by Thykof** [comment]
guys you have removed routeFor for search and store ^^ please wait for a second approval to merge to avoid rework ;) I'm fixing it

**Comment by Thykof** [comment]
you also removed window.open on click on Launch button

**Pull request merged**
